user/src/main/java/org/tessell/widgets/StubRowTable.java

```diff
@@ -90,4 +92,14 @@ public void add(int index, IsWidget a) {
       insertRow(index, a);
     }
   }
+
+  @Override
+  public int indexOfRow(Widget row) {
+    return rows.indexOf(row);
+  }
+
+  @Override
+  public int indexOfRow(com.google.gwt.user.client.ui.IsWidget row) {
+    return rows.indexOf(row.asWidget());
```

Comment:
In a unit test, row.asWidget() will blow up saying "i'm not a real widget". And rows is a collection of IsWidgets anyway, so I think just rows.indexOf(row) is what you want.